### PR TITLE
Maintenance: add optional `now` positional param to getMaintenanceScore

### PR DESCRIPTION
**Pull request by kevmoo**
Allows testing
Also added tests for Maintenance

### CODE REVIEW

File: lib/src/maintenance.dart [hunk 1]
```diff
@@ -100,8 +100,8 @@ class Maintenance extends Object with _$MaintenanceSerializerMixin {
   factory Maintenance.fromJson(Map<String, dynamic> json) =>
       _$MaintenanceFromJson(json);
 
-  double getMaintenanceScore(DateTime publishDate) {
-    final now = new DateTime.now().toUtc();
+  double getMaintenanceScore(DateTime publishDate, {DateTime now}) {
```
Comment (isoos) [inline lib/src/maintenance.dart]: Would be to have `Duration age` as the only parameter?

Comment (kevmoo) [reply lib/src/maintenance.dart]: Ha! that's next

Comment (kevmoo) [reply lib/src/maintenance.dart]: Wanted to avoid breaks until we got good tests...

LGTY?

Comment (isoos) [reply lib/src/maintenance.dart]: please add a TODO there

**Pull request merged**